packages/jotai-ai/package.json

```diff
@@ -30,18 +31,27 @@
   "author": "himself65 <himself65@outlook.com>",
   "license": "MIT",
   "devDependencies": {
+    "@ai-sdk/provider": "^0.0.26",
```

Comment:
should be in `dependencies`

Reply:
Ok.

A small question, I just checked it that `@ai-sdk/provider` and `ai` currently is only used and imported in type definition (`import type ... from ...`). Do I still need to move to `dependencies`?